src/main/java/com/overops/plugins/sonar/util/SimpleUrlClient.java

```diff
@@ -0,0 +1,111 @@
+package com.overops.plugins.sonar.util;
+
+import com.google.common.base.Strings;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+import com.takipi.api.core.consts.ApiConstants;
+import com.takipi.api.core.request.intf.ApiPostRequest;
+import com.takipi.api.core.result.intf.ApiResult;
+import com.takipi.api.core.url.UrlClient;
+import com.takipi.common.util.Pair;
+
+import java.net.HttpURLConnection;
+import java.util.Map;
+
+public class SimpleUrlClient extends UrlClient {
```

Comment:
comment about why we need this?

Reply:
I create this class to be a simple client/request to some API where we put a URL and get a response.
RemoteApiClient - is dedicated to OverOps API calls and has a constructor with package visibility
Therefore I write my own based on UrlClient. Do we need to make it base on BaseApiClient